src/sinks/local-storage.js

```diff
@@ -1,13 +1,9 @@
 export default function localStorageSink(todosData) {
+
   // Observe all todos data and save them to localStorage
   let savedTodosData = {
-    list: todosData.list.map(todoData =>
-      ({
-        title: todoData.title,
-        completed: todoData.completed,
-        id: todoData.id
-      })
-    )
+    list: todosData.get('list').filter(x => x ).toJS()
```

Comment:
Sometimes I got `null` values in the list. Don't know if this was a testing issue or something about immutable.js I haven't discovered yet..
That's why I filter the values here ...